primitives/avail/src/header/extension/v2.rs

```diff
@@ -13,11 +13,12 @@ use crate::{asdr::DataLookup, v2::KateCommitment};
 pub struct HeaderExtension {
 	pub commitment: KateCommitment,
 	pub app_lookup: DataLookup,
+	pub data_root: Option<H256>
```

Comment:
We need to remove the `Option` to facilitate the reading of fixed size from the end.
`app_lookup` should be before `commitment` too, right?